packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -566,10 +566,10 @@ describe('src/cypress/dom/visibility', () => {
         cy.$$('body').css('display', 'none')
 
         expect(cy.$$('html')).not.to.be.hidden
-        expect(cy.$$('html')).to.be.visible
+        // expect(cy.$$('html')).to.be.visible
 
         expect(cy.$$('body')).not.to.be.hidden
-        expect(cy.$$('body')).to.be.visible
+        // expect(cy.$$('body')).to.be.visible
```

Comment:
@senpl Can you explain the reason for this change?

Reply:
Most users don't check visibility of html and body. Also discussion is html is visible is not true false answer(some part are for sure not visible). Also it would be good to be consistent with what users get in browser checkVisibility and in other frameworks. Of course it could stay as it was before. But I guess now it's as close to other frameworks as possible.

Reply:
>Most users don't check visibility of html and body.

Where does this assumption come from?

We would like to not introduce any breaking changes. So keep the tests having the same result as before when checking the html and body visibility. Otherwise this entire PR would be blocked until we do a breaking change release.

Reply:
> Where does this assumption come from?
Typical cypress workflow which could be seen in many repos e.g. on GitHub not include many body and html checks.

Reply:
I added. Still it's now two additional checks just to be consistent with old not useful test.

Reply:
@jennifer-shehane I strongly recommend delete this whole test. Users clearly sets display none and we expect to element to not be hidden. But in reality it is hidden. So we create alternative reality just because it worked like that in past.
Performance would also be better without two useless checks that are just to make this test pass.